Nef_3/include/CGAL/Nef_3/SNC_external_structure.h

```diff
@@ -123,21 +123,53 @@ int sign_of(const CGAL::Plane_3<R>& h)
 }
 
 struct Plane_lt {
-  template <typename R>
-  bool operator()(const CGAL::Plane_3<R>& h1,
-                  const CGAL::Plane_3<R>& h2) const
-  { 
-    typedef typename R::RT     RT;
-    RT diff = h1.a()-h2.a();
-    if ( (diff) != 0 ) return CGAL_NTS sign(diff) < 0;
-    diff = h1.b()-h2.b();
-    if ( (diff) != 0 ) return CGAL_NTS sign(diff) < 0;
-    diff = h1.c()-h2.c();
-    if ( (diff) != 0 ) return CGAL_NTS sign(diff) < 0;
-    diff = h1.d()-h2.d(); return CGAL_NTS sign(diff) < 0;
-  }
+    template <typename R>
+    bool operator()(const CGAL::Plane_3<R>& h1,
+                    const CGAL::Plane_3<R>& h2) const
+    {
+        typedef typename R::RT     RT;
+        bool flag=false;
+        RT ratioa,ratiob,ratioc,ratiod;
```

Comment:
`RT` means ring type. There is no division.

Use the following:
```C++
typedef typename R::FT     FT;
FT ratioa,ratiob,ratioc,ratiod;
```
it should work.
For the variable `RT diff`, [line 159][l159], I think you can stay with `RT`.